### PR TITLE
bump to 2.1.0

### PR DESCRIPTION
**Pull request by bgsach**
Bump to 2.1.0 in preparation for release

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name = "qat-compiler"
 # This name has the -compiler suffix in order to use the poetry and twine tools to build and publish to PyPI
 # witout having to manually adjust the dist file names.
-version = "2.0.3"
+version = "2.1.0"
```
Comment (keriksson-rosenqvist) [inline pyproject.toml:5]: Not critical but we should adjust the authors list

**Review by keriksson-rosenqvist — APPROVED** [review]
LGTM, though as I've remembered is recently, could we also bump version etc. in the Sphinx doc settings please.

**Comment by bgsach** [comment]
https://github.com/oqc-community/qat/pull/190/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711L53 - @keriksson-rosenqvist is this expected?

File: pyproject.toml (hunk 1 above)
Comment (bgsach) [reply pyproject.toml:5]: Done (assuming I didn't miss anyone)

**Comment by bgsach** [comment]
> LGTM, though as I've remembered is recently, could we also bump version etc. in the Sphinx doc settings please. https://github.com/oqc-community/qat/blob/main/docs/source/conf.py#L20

Done

**Comment by bgsach** [comment]
> https://github.com/oqc-community/qat/pull/190/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711L53 - @keriksson-rosenqvist is this expected?

OK, I understand why this is necessary.

File: pyproject.toml [hunk 2]
```diff
@@ -14,6 +14,9 @@ authors = [
     "Kajsa Eriksson Rosenqvist <keriksson.rosenqvist@oxfordquantumcircuits.com>",
     "Norbert Deak <ndeak@oxfordquantumcircuits.com>",
     "Owen Arnold <oarnold@oxfordquantumcircuits.com>",
+    "Benjamin sach <bsach@oxfordquantumcircuits.com>",
```
Comment (keriksson-rosenqvist) [inline pyproject.toml:17]: Capital S?

File: docs/source/conf.py [hunk 3]
```diff
@@ -16,8 +16,11 @@
     "Kajsa Eriksson Rosenqvist <keriksson.rosenqvist@oxfordquantumcircuits.com>, "
     "Norbert Deak <ndeak@oxfordquantumcircuits.com>, "
     "Owen Arnold <oarnold@oxfordquantumcircuits.com>, "
+    "Benjamin sach <bsach@oxfordquantumcircuits.com>, "
+    "Daria Van Hende <dvanhende@oxfordquantumcircuits.com>, "
+    "Luke Causer <lcauser@oxfordquantumcircuits.com>"
 )
-release = version = "1.0.0"
+release = version = "2.1.0"
```
Comment (keriksson-rosenqvist) [inline docs/source/conf.py:23]: :+1:

**Pull request merged**